docs/source/fundamentals/tips.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Tips for success
+description: Your starting point for learning GraphQL and Apollo
+---
+
+TK WIP TODO finish this intro.  Depending on the size of an organization, there are usually existing practices surrounding the management of the data 
```

Comment:
For this intro, I think we want to instill confidence that we are the right people to guide teams on their GraphQL journey and that we've established these best practices based on years of experience from running GraphQL in production